### ISSUE TITLE
about resnet12-pth

### ISSUE THREAD

**Issue by Crazek-CZA** [RongKaiWeskerMA/INSTA#3]
I'm a student from Nanjing University. I'm trying to reproduce your paper. Can you send me the pre-training parameters of resnet12 used in training?
^_^

**Comment by RongKaiWeskerMA** [comment]
Hi, thanks for being interested in this work. I am sorry that I currently, don't have the pretrained model at hands. However, the pretrained resnet-12 can be easily obtained by using the pretrained code from [FEAT](https://github.com/Sha-Lab/FEAT/tree/master).

**Comment by SuzukazeSasagawa** [comment]
Could you tell us the accuracy of INSTA-ProtoNet model and ResNet12 backbone without pre-train parameters?
